astro/src/content/docs/get-started/download-and-install/system-requirements.mdx

```diff
@@ -30,7 +30,13 @@ Docker, k8s and other container platforms are supported because the host operati
 
 FusionAuth ships with its own version of Java. The version that is contained in the bundle is the required and supported version.
 
-Currently, Java 17 is the supported major version. Java 14 is the supported major version for all FusionAuth releases before FusionAuth 1.32.
+Currently, Java 21 is the supported major version. The versions of Java that are supported for prior versions of FusionAuth are:
+
+| FusionAuth Version | Java Version |
+|--------------------|--------------|
+| 1.53.0 and newer   | Java 21      |
+| 1.32.0 - 1.52.1    | Java 17      |
+| 1.31.0 and earlier | Java 11      |
```

Comment:
We've never used Java 11 in the field. We went from `8` -> `14` -> `17` -> `21`.

Reply:
that's what I meant!